obd/obd.py

```diff
@@ -207,13 +213,46 @@ def print_commands(self):
         for c in self.supported_commands:
             print(str(c))
 
-
+    def print_discovered(self):
+        """
+            Utility function meant to print all information discovered:
+            protocol, port name, port baudrate and all supported commands.
+        """
+        if self.interface is not None:
+            print ("The following settings were used to connect to the ECU:")
+            print ("Protocole: " + self.get_protocol_name())
+            print ("Port name: " + self.get_port_name())
+            print ("Port rate: " + self.get_port_baudrate())
+            print ("The following OBD commands are supported:")
+
+            _mylist=[]
+            for c in self.supported_commands:
+                _mylist.append(str(c))
+            _mylist.sort()
+            for i in _mylist:
+                print (i)
+            
+        else:
+            print ("Impossible to print discovered information: no connection to the ECU.")
+            
```

Comment:
Would you mind rebasing-out these changes? (assuming that they aren't co-dependant) That way, the two changes can cleanly come in via two PR's. Same with the addition of the `get_` prefixes; Discuss that separately.